client/src/ui/track/NewApplicationModal.tsx

```diff
@@ -17,14 +19,14 @@ interface NewApplicationModalProps {
   existingApplication?: Application;
 }
 
-const jobTypeOptions = ["Full-time", "Part-time", "Contract", "Internship", "Co-Op", "New Grad"]; // This is temporary
+const jobTypeOptions = ["Internship", "New Grad", "Co-Op", "Full-time", "Part-Time", "Contract"];
```

Comment:
Use the enumerated type "OpportunityType" from the database to keep consistency

Reply:
fixed!